config/application.rb

```diff
@@ -17,6 +17,17 @@
 
 module RedrumNursery
   class Application < Rails::Application
+    config.action_mailer.delivery_method = :smtp
+
+    config.action_mailer.smtp_settings = {
+      address: 'smtp.mandrillapp.com',
+      port:    '587',
+      domain:  'ochotickets.herokuapp.com',
+      user_name: 'SamsonLB@gmail.com',
+      password: '_Tye00aoJEvLcE6fsJEpJg',
+      authentication: 'plain',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.